flutter-idea/src/io/flutter/run/MainFile.java

```diff
@@ -79,11 +79,15 @@ public boolean hasFlutterImports() {
    * If there is an error, {@link Result#canLaunch} will return false and the error is available via {@link Result#getError}
    */
   @NotNull
-  public static MainFile.Result verify(@Nullable String path, Project project) {
+  public static MainFile.Result verify(@Nullable String path, @Nullable Project project) {
     if (!ApplicationManager.getApplication().isReadAccessAllowed()) {
       throw new IllegalStateException("need read access");
     }
 
+    if (project == null) {
```

Comment:
typically we also check for project.isDisposed()

Reply:
Interesting! It sure would be an odd case if a project was disposed while you were trying to run it but not impossible. Thanks!